### PR TITLE
Fixes relating to Proton & the Steam Runtime

### PR DESCRIPTION
**Pull request by CommandMC**
---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [X] Tested the feature and it's working on a current and clean install.
- [X] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Review by flavioislima — APPROVED** [review]
Looks good.
Tested here and its working fine 👍🏽

File: electron/launcher.ts [hunk 1]
```diff
@@ -259,7 +259,7 @@ function setupWineEnvVars(gameSettings: GameSettings, gameId = '0') {
   if (gameSettings.enableResizableBar) {
     ret.VKD3D_CONFIG = 'upload_hvv'
   }
-  if (gameSettings.useSteamRuntime) {
+  if (wineVersion.type === 'proton') {
```
Comment (flavioislima) [inline electron/launcher.ts:262]: Ah one question:
dont we need that for the steam runtime for native games?

Comment (CommandMC) [reply electron/launcher.ts:262]: No, these settings are only relevant for Proton
The function (`setupWineEnvVars`) shouldn't even be called for native games

Comment (flavioislima) [reply electron/launcher.ts:262]: I see 👍🏽

**Pull request merged**